Gruntfile.js

```diff
@@ -38,12 +38,26 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['clean:dist', 'sass', 'webpack']);
 
+  // Regular grunt server, used when running inside the payments-env docker.
   grunt.registerTask('serve',
-    ['clean:dist', 'sass:dev', 'webpack-dev-server:start', 'watch:sass']);
-  grunt.registerTask('start',
     ['clean:dist', 'sass:dev', 'webpack:dev', 'watch:sass']);
-  grunt.registerTask('start-email',
+  // The dev server used for live reloading in the browser.
+  grunt.registerTask('serve:dev',
```

Comment:
Aha, I didn't realise that syntax was for multi tasks, my bad.

`watch` feels a *lot* better than `start`, I would go for that over my suggestions.

I also like the tables in the README idea. I'll fix this PR up to reflect these ideas and push a new version.